ehitchhiking-ui/src/app/shared/modals/profile-modal/car-info-form/car-info-form.component.ts

```diff
@@ -23,8 +22,7 @@ export class CarInfoFormComponent implements OnInit {
 		this.addCarForm = this.formBuilder.group({
 			model: ['', [Validators.required, Validators.pattern('^[0-9a-zA-Z-]{3,50}$')]],
 			color: ['', [Validators.required, Validators.pattern('^[a-zA-Z-]{3,20}$')]],
-			carNumber: ['', [Validators.required, Validators.pattern('^[0-9a-zA-Z-]{6,8}$')]],
-			experience: ['', [Validators.required, Validators.pattern('^[0-9]{1,2}$'), Validators.max(80)]],
```

Comment:
Do you have the customer's approval to remove experience input?

Reply:
yes, we have